### ISSUE TITLE
Color Font looks a bit light in v1.2018.10.19-beta4

### ISSUE THREAD

**Issue by vimcaw** [snowie2000/mactype#468]
Color Font support is a wonderful featue, thanks the maintainers.

The only thing that is insufficient, after I try it, I think it looks a bit light.

MacType | DirectWrite
------------ | -------------
![image](https://user-images.githubusercontent.com/21964227/47835470-0835f700-dddf-11e8-8861-d92c6c0bf794.png) | ![image](https://user-images.githubusercontent.com/21964227/47835494-24399880-dddf-11e8-95bb-6f54fcf1a2d4.png)
![image](https://user-images.githubusercontent.com/21964227/47835626-e5581280-dddf-11e8-9768-361b3971b250.png) | ![image](https://user-images.githubusercontent.com/21964227/47835637-f56ff200-dddf-11e8-8c4a-05d6c333ea6d.png)
![image](https://user-images.githubusercontent.com/21964227/47835716-50094e00-dde0-11e8-8899-d467957dddb7.png) | ![image](https://user-images.githubusercontent.com/21964227/47835731-5e576a00-dde0-11e8-98e8-9be4a1f18827.png)
![image](https://user-images.githubusercontent.com/21964227/47835784-98c10700-dde0-11e8-9914-19793f85a925.png) | ![image](https://user-images.githubusercontent.com/21964227/47835795-a24a6f00-dde0-11e8-8202-8f2c5e23adf2.png)

## MacType info: 
* MacType version: v1.2018.10.19-beta4

Relevant MacType config: 
```ini
[experimental]
;Color font support
colorfont=1

;To make colors appear correctly in Chrome
invertcolor=1
```

## System info:

* Windows 10 Home
* version: 1809
* OS version: 17763.55

**Comment by vimcaw** [comment]
Ok, understood, it's normal in other apps.

Notepad: 
![image](https://user-images.githubusercontent.com/21964227/47837854-7337fb80-dde8-11e8-9b52-b483375c36c0.png)
Chrome (After commenting the `invertcolor`): 
![image](https://user-images.githubusercontent.com/21964227/47838180-8a2b1d80-dde9-11e8-950e-0eacc6c9c217.png)

**Comment by sammilucia** [comment]
Is that why the arrays in https://github.com/snowie2000/mactype/commit/8f8896a5c96e03eb9446a8311167e431dd71e44b are all crazy?

```
unsigned char InvertRed[256] = { 255, 254, 253, 252, 251, 250, 249, 248, 247, 246, 245, 244, 243, 242, 241, 240, 239, 239, 238, 237, 236, 235, 234, 233, 232, 231, 230, 229, 228, 227, 226, 225, 224, 223, 222, 221, 220, 219, 218, 217, 216, 215, 214, 213, 212, 211, 210, 209, 208, 208, 207, 206, 205, 204, 203, 202, 201, 200, 199, 198, 197, 196, 195, 194, 193, 192, 191, 190, 189, 188, 187, 186, 185, 184, 183, 182, 181, 180, 179, 178, 177, 177, 176, 175, 174, 173, 172, 171, 170, 169, 168, 167, 166, 165, 164, 163, 162, 161, 160, 159, 158, 157, 156, 155, 154, 153, 152, 151, 150, 149, 148, 147, 146, 146, 145, 144, 143, 142, 141, 140, 139, 138, 137, 136, 135, 134, 133, 132, 131, 130, 129, 128, 127, 126, 125, 124, 123, 122, 121, 120, 119, 118, 117, 116, 115, 115, 114, 113, 112, 111, 110, 109, 108, 107, 106, 105, 104, 103, 102, 101, 100, 99, 98, 97, 96, 95, 94, 93, 92, 91, 90, 89, 88, 87, 86, 85, 84, 84, 83, 82, 81, 80, 79, 78, 77, 76, 75, 74, 73, 72, 71, 70, 69, 68, 67, 66, 65, 64, 63, 62, 61, 60, 59, 58, 57, 56, 55, 54, 53, 53, 52, 51, 50, 49, 48, 47, 46, 45, 44, 43, 42, 41, 40, 39, 38, 37, 36, 35, 34, 33, 32, 31, 30, 29, 28, 27, 26, 25, 24, 23, 22, 22, 21, 20, 19, 18, 17, 16, 15, 14, 13, 12, 11, 10, 9, 8 };
unsigned char InvertGreen[256] = { 255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,254,253,252,251,250,249,247,246,245,244,243,242,241,240,239,237,236,235,234,233,232,231,230,228,227,226,225,224,223,222,221,220,218,217,216,215,214,213,212,211,209,208,207,206,205,204,203,202,201,199,198,197,196,195,194,193,192,190,189,188,187,186,185,184,183,181,180,179,178,177,176,175,174,173,171,170,169,168,167,166,165,164,162,161,160,159,158,157,156,155,154,152,151,150,149,148,147,146,145,143,142,141,140,139,138,137,136,134,133,132,131,130,129,128,127,126,124,123,122,121,120,119,118,117,115,114,113,112,111,110,109,108,107,105,104,103,102,101,100,99,98,96,95,94,93,92,91,90,89,88,86,85,84,83,82,81,80,79,77,76,75,74,73,72,71,70,68,67,66,65,64,63,62,61,60,58,57,56,55,54,53,52,51,49,48,47,46,45,44,43,42,41,39,38,37,36,35,34,33,32,30,29,28,27,26,25,24,23,22,20,19,18,17,16,15,14,13,11,10,9,8,7,6,5,4,2,1,0,0,0,0,0,0};
unsigned char InvertBlue[256] = { 255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,254,252,251,250,249,248,247,246,245,244,243,242,240,239,238,237,236,235,234,233,232,231,230,228,227,226,225,224,223,222,221,220,219,218,216,215,214,213,212,211,210,209,208,207,206,204,203,202,201,200,199,198,197,196,195,194,192,191,190,189,188,187,186,185,184,183,182,180,179,178,177,176,175,174,173,172,171,170,168,167,166,165,164,163,162,161,160,159,157,156,155,154,153,152,151,150,149,148,147,145,144,143,142,141,140,139,138,137,136,135,133,132,131,130,129,128,127,126,125,124,123,121,120,119,118,117,116,115,114,113,112,111,109,108,107,106,105,104,103,102,101,100,99,97,96,95,94,93,92,91,90,89,88,87,85,84,83,82,81,80,79,78,77,76,74,73,72,71,70,69,68,67,66,65,64,62,61,60,59,58,57,56,55,54,53,52,50,49,48,47,46,45,44,43,42,41,40,38,37,36,35,34,33,32,31,30,29,28,26,25,24,23,22,21,20,19,18,17,16,14,13,12,11,10,9,8,7,6,5,4,2,1,0,0,0,0,0,0 }; 
```

**Comment by snowie2000** [comment]
Exactly. I painted a lot of emojis and collected all the corresponding data, then I did a formula fitting to all the data collected.

**Comment by sammilucia** [comment]
I'm glad, I was wondering if you'd gone mad 😂

**Comment by snowie2000** [comment]
Alright, I figured out I made a mistake on formula fitting.
I've never thought that a text with a solid color will be blended with the background, but Chrome did.
Last time I did the sample collection, I didn't notice that the web page I used to capture colors has a gray background which heavily affected the result.

I did it again, and now it's much better and looks like the real deal.
![qq 20181102104743](https://user-images.githubusercontent.com/11767189/47891292-9963a700-de8d-11e8-8844-4ffa02790df0.png)

Plus, this is what my formula looks like in the Excel together with data I collected.
![qq 20181102105455](https://user-images.githubusercontent.com/11767189/47891330-bef0b080-de8d-11e8-8bff-1c58d17ff229.png)

Hope someone can give me a better formula of it.
